src/storekit.cs

```diff
@@ -617,11 +647,13 @@ interface SKCloudServiceController {
 		NSString StorefrontCountryCodeDidChangeNotification { get; }
 	}
 
+#if !WATCH
```

Comment:
`UIViewController` is not available on Watch

Reply:
You can create an empty interface for UIViewController at the top under Watch, then use NoWatch, once it available (if ever) you can find all the interfaces in the same place.